index.html

```diff
@@ -812,6 +851,29 @@
                     button.appendChild(titleName);
                 });
 
+                /*
+                 * @jiteshjha, @pooja
+                 * rotateCustom(@parameters) is defined in 'orientation.js'
+                 */
+                editor.addAction('rotateCustom', function(editor, cell) {
```

Comment:
All three actions have the same shortcut! IMHO, you may remove the last three parameters - "null, null, 'Ctrl+R'"